config/filter.d/postfix-rbl.conf

```diff
@@ -12,7 +12,7 @@ before = common.conf
 
 _daemon = postfix(-\w+)?/smtpd
 
-failregex = ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: 454 4\.7\.1 Service unavailable; Client host \[\S+\] blocked using .* from=<\S*> to=<\S+> proto=ESMTP helo=<\S*>$
+failregex = ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: [45]54 [45]\.7\.1 Service unavailable; Client host \[\S+\] blocked using \b
```

Comment:
`\b` makes sense only at word end (not at word begin), because marked end of word.
```diff
- blocked using \b
+ blocked using\b
```
In my previously provided example, even shorter "Service unavailable; Client host example.com blocked" will matched, also if "using something else ..." will be replaced tomorrow with something whole different or missing at all. 
In your not (also if it ends with space).

So either remove `\b` but leave space (or replace with `blocked using\s`), or use expression from my example.